contents/handbook/engineering/feature-ownership.md

```diff
@@ -23,6 +23,7 @@ You can also view the list [directly in GitHub](https://github.com/PostHog/posth
 | Annotations | [Team Product Analytics][Team Product Analytics]  | <span class="lemon-tag gh-tag">feature/annotations</span> |
 | API Structure | Security + core updates owned by [Team Pipeline][Team Pipeline]. Features owned by the relevant small team | <span class="lemon-tag gh-tag">feature/api-structure</span> |
 | Async migrations | [Team Pipeline][Team Pipeline]  | <span class="lemon-tag gh-tag">feature/async-migrations</span> |
+| Batch exports | [Team CDP][Team CDP]  | <span class="lemon-tag gh-tag">feature/cdp</span> |
```

Comment:
data warehouse not cdp and while you're at it you can add data warehouse itself too

Reply:
Doh! Time for more coffee.  Thanks, on it...